### ISSUE TITLE
Show "Device Access" badge for "Fire / Hardware Access" part on slides.html5rocks.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#484]
```
What steps will reproduce the problem?
1. open http://slides.html5rocks.com/#hardware-access-title

What is the expected output? What do you see instead?
The slide shows "Performance" badge for this part.



```

Original issue reported on code.google.com by `han...@google.com` on 8 Mar 2011 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision: 4a5f941264 
(http://code.google.com/p/html5rocks/source/detail?r=4a5f941264540f137956561c1a3
fcf3de8903219)
```

Original comment by `han...@google.com` on 8 Mar 2011 at 10:30
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
LGTM
```

Original comment by `ericbide...@html5rocks.com` on 8 Mar 2011 at 5:47
